### PR TITLE
Add the new added virtio node

### PR DESCRIPTION
**Pull request by yhe39**

### CODE REVIEW

File: drm/ResourceManager.h [hunk 1]
```diff
@@ -78,6 +79,7 @@ class ResourceManager {
   PipelineToFrontendBindingInterface *const frontend_interface_;
 
   bool initialized_{};
+  int card_num;
```
Comment (feijiang1) [inline drm/ResourceManager.h:82]: card_num should be card_num_

File: drm/ResourceManager.cpp [hunk 2]
```diff
@@ -66,6 +66,36 @@ static int find_virtio_gpu_card(ResourceManager *res_man, char* path_pattern, in
...
+    std::ostringstream path;
+    path << path_pattern << idx;
+
+    struct stat buf {};
+    if (stat(path.str().c_str(), &buf) != 0)
+      break;
+
+    node_num++;
+  }
+  if (node_num > card_num) {
```
Comment (feijiang1) [inline drm/ResourceManager.cpp:87]: currently we assume the new added card is ivshmem device, can we call ioctl to double confirm that.
Also need add more description for these code.

File: drm/ResourceManager.cpp [hunk 3]
```diff
@@ -66,6 +66,36 @@ static int find_virtio_gpu_card(ResourceManager *res_man, char* path_pattern, in
...
+    card_num = node_num;
+    std::ostringstream path;
+    path << path_pattern << (node_num - 1);
+    auto dev = DrmDevice::CreateInstance(path.str(), this);
+    if (dev) {
+      ALOGE("create ivshmem node card\n");
```
Comment (feijiang1) [inline drm/ResourceManager.cpp:93]: ALOGE=>ALOGD

File: drm/ResourceManager.cpp (hunk 2 above)
Comment (yhe39) [reply drm/ResourceManager.cpp]: The hwc use the card, while the minigbm open the renderD, i am not sure whether in hwc the card could query the device info like renderD in minigbm.
@phreer , do you know that?

Comment (yhe39) [reply drm/ResourceManager.cpp]: Check the node is ivshm device logic is addede.
Also i add the reloaded_ to make sure android only open one ivshm device.

**Review by feijiang1 — APPROVED** [review]
LGTM

**Comment by feijiang1** [comment]
@HaihongxLi please help review together

**Review by feijiang1 — APPROVED** [review]
LGTM

**Pull request merged**